aten/src/ATen/core/VariableFallbackKernel.cpp

```diff
@@ -60,9 +62,44 @@ TORCH_LIBRARY_IMPL(_, AutogradMLC, m) {
   m.fallback(torch::CppFunction::makeFallthrough());
 }
 
+namespace {
+  void functionalizeFallback(const c10::OperatorHandle& op, c10::DispatchKeySet dispatchKeySet, torch::jit::Stack* stack) {
+    const auto& schema = op.schema();
+    const auto num_arguments = schema.arguments().size();
+    const auto arguments_begin = stack->size() - num_arguments;
+    auto arguments = torch::jit::last(stack, num_arguments);
+
+    for (int64_t idx = 0; idx < num_arguments; ++idx) {
+      const auto& ivalue = arguments[idx];
+      if (ivalue.isTensor()) {
+        at::Tensor t = ivalue.toTensor();
+		at::functionalization::impl::maybe_sync(t);
+      } else if (ivalue.isTensorList()) {
+        auto tensors = ivalue.toTensorList();
+		at::functionalization::impl::maybe_sync(tensors);
```

Comment:
the `at::Tensor t` and `auto tensors` take out owning references which is totally unnecessary here